include/Evaluation_und_Validation.tex

```diff
@@ -1,5 +1,47 @@
 \chapter{Evaluation und Validation}
 \label{ch:Eval}
-
+In diesem Kapitel wird evaluiert, ob das Projekt und dessen Produzierte Artefakte die Anforderungen erfüllen. Dazu wurden alle Anforderungen nochmals zusammengetragen und jede einzelne mit dem jeweiligen Artefakt überprüft, das Resultat wurde schliesslich in forme einer einfach lesbaren Tabelle festgehalten. Aus dem Vergleich der konnte schliesslich festgestellt werden, dass alle priorisierten Anforderungen erfüllt werden konnten. Daher kann dieses Projekt als erfolgreich betrachtet werden.
```

Comment:
[...], ob das Projekt und dessen _produzierten_ Artefakte die Anforderungen erfüllen. [...], das Resultat wurde schliesslich in _Form_ einer eingach lesbaren Tabelle festgehalten.